CHANGELOG.unreleased.md

```diff
@@ -34,3 +37,4 @@ Section headings should be at level 3 (e.g. `### Added`).
 
 - Fixed ValueError on Windows when running a W&B script from a different drive (@jacobromero in https://github.com/wandb/wandb/pull/9678)
 - Fix base_url setting was not provided to wandb.login (@jacobromero in https://github.com/wandb/wandb/pull/9703)
+- Fix `IsADirectoryError` when providing a path to a directory when creating a `wandb.html` object (@jacobromero in https://github.com/wandb/wandb/pull/9728)
```

Comment:
For these ones, you don't need to say "Fix" because they're already in the Fixed section. They should just be sentences describing the new state of things in contrast to the old state, like "`wandb.Html()` no longer raises an error when the given string matches a path to an existing directory".